### PR TITLE
Update README, add CONTRIBUTING

### PR DESCRIPTION
**Pull request by wvanbergen**
@girasquid another one :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Include the version of OffsitePayments, Ruby, ActiveSupport, and Nokogiri you are using.
+
+### Pull request guidelines
+
+1. [Fork it](http://github.com/Shopify/offsite_payments/fork) and clone your new repo
+2. Create a branch (`git checkout -b my_awesome_feature`)
+3. Commit your changes (`git add my/awesome/file.rb; git commit -m "Added my awesome feature"`)
+4. Push your changes to your fork (`git push origin my_awesome_feature`)
+5. Open a [Pull Request](https://github.com/shopify/offsite_payments/pulls)
+
```
Comment (girasquid) [inline CONTRIBUTING.md]: Do you need the second newline?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Your code should support all the Ruby versions and ActiveSupport versions we have enabled on Travis CI.
+- No new gem dependencies will be accepted.
+- **XML**: use Nokogiri for generating and parsing XML.
+- **JSON**: use `JSON` in the standard library to parse and generate JSON.
+- **HTTP**: use `ActiveUtils::PostsData` to do HTTP requests.
+- Do not update the CHANGELOG, or the `OffsitePayments::VERSION` constant.
+
+### Placement within Shopify
+
+The addition of your payment integration to offsite_payments does not guarantee placement within Shopify. In order to have your gateway considered, please send an email to payment-integrations@shopify.com with **Offsite_Paymebt Integration** in the subject. Be sure to include:
```
Comment (girasquid) [inline CONTRIBUTING.md]: Typo

**Comment by girasquid** [comment]
Two minor niggles, otherwise :+1:

**Comment by wvanbergen** [comment]
Both are fixed. :)

**Comment by girasquid** [comment]
:heart_eyes:

**Pull request merged**